src/Avatar/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
Why does the badge branch contains avatar file changes?

Reply:
the badge component uses on avatar component for the gray circle

Reply:
You should branch from develop in this case, the branch should ideally contain badge related changes to minimize conflicts and have cleaner branch tree